packages/typescript/package.json

```diff
@@ -12,7 +12,7 @@
     "@theia/monaco": "^0.4.0",
     "@theia/workspace": "^0.4.0",
     "command-exists": "^1.2.8",
-    "typescript-language-server": "^0.3.7"
+    "typescript-language-server": "0.4.0-dev.12aa8ed"
```

Comment:
@kittaakos  `0.4.0-dev.ce6b943` works better now:

<img width="433" alt="screen shot 2019-03-06 at 15 09 45" src="https://user-images.githubusercontent.com/914497/53887141-edd88800-4021-11e9-8e47-0c1a945678b4.png">

Reply:
Nice, can you please push to this PR? Can be force push too.